### ISSUE TITLE
Issue with pushState

### ISSUE THREAD

**Issue by mattdesl** [bigwheel-framework/bigwheel#35]
I have the following:

```
module.exports = {
  '/': require('../sections/landing/index.js'),
  '/mount': require('../sections/mount/index.js'),
  pushState: false
};
```

I open my site at `localhost:9966/` (with budo).

If I do `framework.go('/mount')`, it will animate to the mount page and the url will be `#!/mount`. If I then hit the browser back button, it will take me back to `#!/` and show the landing. Then, I do another `framework.go('/mount')`, but the mount section is not getting `init` called.

However, everything works if the first time I open my site, the url is `localhost:9966/#!/`. To work around this I am currently doing this before framework init:

```
if (window.location.hash === '') {
  window.location.href = '/#!';
}
```

**Comment by mikkoh** [comment]
@njam3 I wonder if this issue was introduced after the `location-bar` implementation in f1-router.

**Comment by njam3** [comment]
@mattdesl @mikkoh This bug should be fixed now as of `bw-router@2.0.5`
